releasenotes/notes/v1.0.0-RC2-v0.68.0-otel-f1571b993848d654.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The OTLP ingest endpoint now supports the same settings and protocols as the OpenTelemetry Collector OTLP receiver v1.0.0-RC2/v0.68.0.
```

Comment:
I am inclined to say we should just say `v0.68.0` here, since most users will understand this better (v1.0.0-RC2 is the version of pdata, not of the OTLP receiver)